src/components/calculator.js

```diff
@@ -1,48 +1,130 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import calculate from '../logic/caculate';
```

Comment:
- [ OPTIONAL] Let's capitalize the name of the `Calculator.js` component so as to differentiate it from the imported one `calculate.js`  file from the logic directory and remember to always capitalize your react name component file
- eg: `Navbar.js, Home.js....`

Reply:
Done.